sdks/builds/wasm.mk

```diff
@@ -88,17 +88,19 @@ _wasm-$(1)_CONFIGURE_FLAGS= \
 	--disable-nls \
 	--disable-support-build \
 	--enable-maintainer-mode \
-	--enable-minimal=appdomains,com,remoting
+	--enable-minimal=appdomains,com,remoting \
+	--enable-icall-symbol-map \
+	--with-cross-offsets=wasm32-unknown-none.h
 
 $$(eval $$(call CrossRuntimeTemplate,wasm-$(1),$$(if $$(filter $$(UNAME),Darwin),$(2)-apple-darwin10,$$(if $$(filter $$(UNAME),Linux),$(2)-linux-gnu,$$(error "Unknown UNAME='$$(UNAME)'"))),$(3)-unknown-none,$(4),$(5),$(6)))
 
+ifdef ENABLE_WASM_CROSS
 wasm_TARGETS += wasm-$(1)-$$(CONFIGURATION) $(5)
+endif
 
 endef
 
-ifeq ($(DISABLE_WASM_CROSS),)
```

Comment:
Won’t that force users with cross disabled to provision mxe?

Reply:
Don't think so, this just enables the make targets.